src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -76,8 +81,9 @@ function useChatUtilsHook(): UseChatUtilsHook {
 
     refresh('channelList');
     refresh('channelInfo');
-    refresh('chat');
     refresh('user');
+    refreshWithId('chat', channel?.id);
+    // refresh('chat');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The refresh method now takes an additional parameter 'chat' with the channel id. This seems to be a targeted refresh based on specific IDs which is a good practice for performance optimization.



<!-- This is an auto-generated comment by OSS CodeRabbit -->